services/storage/src/simcore_service_storage/utils/simcore_s3_dsm_utils.py

```diff
@@ -131,6 +137,50 @@ def compute_file_id_prefix(file_id: str, levels: int):
     return "/".join(components[:levels])
 
 
+def get_random_export_name(user_id: UserID) -> StorageFileID:
+    return TypeAdapter(StorageFileID).validate_python(
+        f"{EXPORTS_S3_PREFIX}/{user_id}/{uuid4()}.zip"
```

Comment:
ok so this is indeed a uuid. please adjust the tests accordingly. why they even work?

Reply:
I have changed the regex so it accepts only this format